.github/CODEOWNERS

```
@@ -51,6 +51,7 @@ airflow-core/src/airflow/ui/public/i18n/locales/nl/ @BasPH # + @DjVinnii
 airflow-core/src/airflow/ui/public/i18n/locales/pl/ @potiuk @mobuchowski # + @kacpermuda
 airflow-core/src/airflow/ui/public/i18n/locales/tr/ @bugraoz93 # +@hasancatalgol
 airflow-core/src/airflow/ui/public/i18n/locales/zh-TW/ @Lee-W @jason810496 # + @RoyLee1224 @guan404ming
+airflow-core/src/airflow/ui/public/i18n/locales/zh-CN/ 
```

(1) need to be moved-up one line as well to be "in order".
Empty line w/o codeowner does not help. Would be like:
```suggestion
airflow-core/src/airflow/ui/public/i18n/locales/zh-CN/ @<some-commiter-id> # + @Fortytwoo
airflow-core/src/airflow/ui/public/i18n/locales/zh-TW/ @Lee-W @jason810496 # + @RoyLee1224 @guan404ming
```

---

Thank you for the suggestion, which I had never considered in the past.
It has been corrected.